packages/react/tooltip/src/Tooltip.tsx

```diff
@@ -182,9 +216,9 @@ interface TooltipTriggerProps extends PrimitiveButtonProps {}
 
 const TooltipTrigger = React.forwardRef<TooltipTriggerElement, TooltipTriggerProps>(
   (props: ScopedProps<TooltipTriggerProps>, forwardedRef) => {
-    const { __scopeTooltip, ...triggerProps } = props;
-    const context = useTooltipContext(TRIGGER_NAME, __scopeTooltip);
-    const popperScope = usePopperScope(__scopeTooltip);
+    const { __scopeTooltipProvider, ...triggerProps } = props;
```

Comment:
I'm rather confused by all the renaming here, I'm sure there's a reason but I can't figure it out right now…

Reply:
Heh, I thought this might get raised. The uppermost component is the `TooltipProvider` now so I renamed it to be scoped by the uppermost. I did similar in `Toast` (it's `__scopeToastProvider`). 

I'm happy to change this back to `__scopeTooltip` and do `__scopeToast` for toast instead but it means that I can't define the scope at the top of the file anymore I end up with "Cannot use before defined/assigned" errors no matter where I put it.